layers/meta-balena-beaglebone/recipes-kernel/linux/linux-bb.org_git.bbappend

```diff
@@ -1,7 +1,3 @@
 inherit kernel-balena
 
-# BRANCH:aarch64 = "v5.10.168-ti-arm64-r111"
```

Comment:
These changes from here should be removed from here and also the comments should not be added at all in the previous comment

Reply:
Done.